python/tests/test_tree_stats.py

```diff
@@ -3390,7 +3389,7 @@ def f(x):
             return x * (x < n)
 
         # Determine output_dim of the function
-        for mode in ("site", "branch", "node"):
+        for _mode in ("site", "branch", "node"):
             sigma1 = ts.sample_count_stat(sample_sets, f, 3, windows=windows)
```

Comment:
Definite error, ``mode`` should be an arg to sample_count_stat

Reply:
Putting `mode` as an arg caused the test to fail. Needed changes to `denom` to match the shape of `sigma3`.

Reply:
Good catch!